controllers/propagator/encryption_test.go

```diff
@@ -103,3 +105,32 @@ func TestGetEncryptionKeyCached(t *testing.T) {
 	Expect(err).To(BeNil())
 	Expect(string(key[0])).To(Equal("A"))
 }
+
+func TestGetInitializationVector(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	// Test when the initialization vector is generated
+	annotations := map[string]string{}
+
+	client := fake.NewClientBuilder().Build()
```

Comment:
It doesn't hurt to have this but it's technically not needed since the method being tested doesn't use the client.

Reply:
Copy/paste, anyone? 😜